subprojects/docs/src/docs/userguide/commandLineInterface.adoc

```diff
@@ -0,0 +1,520 @@
+
+----
+❯ gradle :mySubproject:taskName
+❯ gradle mySubproject:taskName
+----
+
+You can also run a task for all subprojects using the task name only. For example, this will run the "test" task for all subprojects when invoked from the root project directory.
+
+----
+❯ gradle test
+----
+
+When invoking Gradle from within a subproject, the project name should be omitted:
+
+----
+❯ cd mySubproject
+❯ gradle taskName
+----
+
+==== Executing multiple tasks
+You can also specify multiple tasks. For example, the following will execute the `compile` and `test` tasks in the order that they are listed on the command-line and will also execute the dependencies for each task.
+
+----
+❯ gradle compile test
+----
+
+[[sec:excluding_tasks_from_the_command_line]]
+==== Excluding tasks from execution
+You can exclude a task from being executed using the `-x` or `--exclude-task` command-line option and providing the name of the task to exclude.
+
+++++
+<figure>
+    <title>Example Task Graph</title>
+    <imageobject>
+        <imagedata fileref="img/commandLineTutorialTasks.png"/>
+    </imageobject>
+</figure>
+++++
+
+++++
+<sample id="excludeTask" dir="userguide/tutorial/excludeTasks" title="Excluding tasks">
+    <output args="dist --exclude-task test"/>
+</sample>
+++++
+
+You can see that the `test` task is not executed, even though it is a dependency of the `dist` task. The `test` task's dependencies such as `compileTest` are not executed either. Those dependencies of `test` that are required by another task, such as `compile`, are still executed.
+
+[[sec:rerun_tasks]]
+==== Forcing Tasks To Execute
+
+You can force Gradle to execute all tasks ignoring up-to-date checks using the `--rerun-tasks` option:
+
+----
+❯ gradle test --rerun-tasks
+----
+
+This will force `test` and _all_ task dependencies of `test` to execute. It's a little like running `gradle clean test`, but without the build's generated output being deleted.
+
+[[sec:continue_build_on_failure]]
+==== Continuing the Build When a Failure Occurs
+
+By default, Gradle will abort execution and fail the build as soon as any task fails. This allows the build to complete sooner, but hides other failures that would have occurred. In order to discover as many failures as possible in a single build execution, you can use the `--continue` option.
+
+----
+❯ gradle test --continue
+----
+
+When executed with `--continue`, Gradle will execute _every_ task to be executed where all of the dependencies for that task completed without failure, instead of stopping as soon as the first failure is encountered. Each of the encountered failures will be reported at the end of the build.
+
+If a task fails, any subsequent tasks that were depending on it will not be executed. For example, tests will not run if there is a compilation failure in the code under test; because the test task will depend on the compilation task (either directly or indirectly).
+
+==== Running Tests
```

Comment:
:o: I feel this rather belongs to the documentation on the Java plugin...at the moment the option is defined by the `Test` task which is completely JVM-based: https://github.com/gradle/gradle/blob/master/subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java#L540-L545

Reply:
Looks like "Running Test" hasn't been migrated and is still part of the "Command Line" chapter.

Reply:
I'll remove it since there's some contention